specs/predict/responses/Segment.yml

```
@@ -108,6 +111,11 @@ segmentName:
   description: The name of the segment.
 
 updatedAt:
+  type: string
+  description: The date and time at which the segment was successfully updated (RFC3339).
```

Is "successfully" needed?

```suggestion
  description: The date and time at which the segment was updated (RFC3339).
```

---

https://github.com/algolia/api-clients-automation/pull/1431/commits/5c3cd7ec0d6c7d10c5011d701e95426f6ac8311f

---

(Discussed on Slack) Yes, you are correct. `updatedAt` is only ever used in the response to a create or update segment request (not used by the pipeline).